Arcade Cloud Infra page and deployment section clarity (#532)
ArcadeAI/docs: app/en/home/hosting-overview/page.mdx

# Hosting Options

The easiest and best way to use Arcade.dev is via our cloud service - sign up for free at [https://api.arcade.dev](https://api.arcade.dev).  However, you might need to connect your tools to local resources (e.g. a local database or filesystem) or keep data within your own infrastructure.  Don't worry, Arcade has you covered via either Arcade Cloud or our on-premise deployment options.

## Arcade Cloud

Arcade Cloud is the default option — sign up and start building immediately:

- **Zero Infrastructure**: No servers or databases to manage
- **Automatic Updates**: Always access the latest tools and features
- **Built-in Scaling**: Handles traffic spikes automatically
- **Free Tier**: Start building without a credit card

### MCP Server Deployment

You can route and manage tool calls from your agents to MCP servers hosted anywhere—on your machine, on your own infrastructure, in a private cloud, or on Arcade's cloud service. This allows you to mix the best public tools with your own private tools.

> Connect on-premises MCP servers to Arcade's cloud engine for a hybrid deployment:

Reviewer: ```suggestion
Connect on-premises MCP servers to Arcade Cloud for a hybrid deployment:
```

- **Private Resources**: Access databases and APIs within your network
- **Data Control**: Keep sensitive data in your environment
- **Custom Dependencies**: Use specific runtime requirements or configurations
- **Compliance**: Meet regulatory requirements while using Arcade's capabilities

> See [On-premise MCP Servers](/home/deployment/on-prem-mcp) for more information about how to use your own MCP servers running anywhere, and see [Arcade Deploy](/home/serve-tools/arcade-deploy) to learn how to deploy to Arcade's cloud.

Reviewer: ```suggestion
See [On-premise MCP Servers](/home/deployment/on-prem-mcp) for more information about how to use your own MCP servers running anywhere, and see [Arcade Deploy](/home/serve-tools/arcade-deploy) to learn how to deploy to Arcade Cloud.
```

### Customizing Auth

> You don't have to self-host Arcade to customize your auth experiences. Arcade's cloud service supports a number of auth providers out of the box, and you can provide your own OAuth app credentials to brand your end-user experience. We recommend doing this for all production use cases, so that you can have isolated rate limits with the OAuth service provider and you can give your users a consistent experience when they go through an auth flow.

Reviewer: ```suggestion
You don't have to self-host Arcade to customize your auth experiences. Arcade Cloud supports a number of auth providers out of the box, and you can provide your own OAuth app credentials to brand your end-user experience. We recommend doing this for all production use cases, so that you can have isolated rate limits with the OAuth service provider and you can give your users a consistent experience when they go through an auth flow.
```
